usr/src/pkg/manifests/service-security-kerberos-5.p5m

```diff
@@ -29,32 +29,29 @@ set name=pkg.description value="Kerberos version 5 support"
 set name=info.classification value=org.opensolaris.category.2008:System/Security
 set name=variant.arch value=$(ARCH)
 dir  path=etc group=sys
-$(not_aarch64)dir path=etc/krb5 group=sys
-$(not_aarch64)file path=etc/krb5/krb5.conf group=sys \
+dir  path=etc/krb5 group=sys
+file path=etc/krb5/krb5.conf group=sys \
```

Comment:
This (and some other bits) come out of cmd/gss, so this breaks packaging, etc.

Reply:
Think this is fixed now. Did the changes to the manifest again.